scripts/install_third_party_libs.py

```
@@ -166,6 +166,47 @@ def install_node() -> None:
+            if common.is_mac_os():
+                node_file_name = (
+                    'node-v%s-darwin-x64' % common.PLAYWRIGHT_NODE_VERSION
+                )
+            elif common.is_linux_os():
+                node_file_name = (
+                    'node-v%s-linux-x64' % common.PLAYWRIGHT_NODE_VERSION
+                )
+            else:
+                raise Exception('Unsupported OS')
+        else:
+            node_file_name = 'node-v%s' % common.PLAYWRIGHT_NODE_VERSION
```

Ruff (0.15.14)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)